pysatNASA/instruments/timed_saber.py

```diff
@@ -99,8 +99,12 @@
                                supported_tags=supported_tags)
 
 # Set the load routine
+# Note that the time variable associated with tpaltitude is renamed to avoid
+# conflict with renaming Epoch.
```

Comment:
```suggestion
# Set the load routine. Note that the time variable associated with
# tpaltitude is renamed to avoid conflict with renaming Epoch.
```